app/lib/bk/compat/parsers/circleci/orbs/docker/lint.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+module BK
+  module Compat
+    module CircleCISteps
+      # Handles the translation of docker installation commands
+      class DockerOrb
+        def translate_docker_hadolint(config)
+          dockerfiles = config.fetch('dockerfiles', 'Dockerfile').split(':')
+          failure_threshold = config.fetch('failure-threshold', 'info')
+          ignore_rules = format_rules(config.fetch('ignore-rules', ''), 'ignore')
+          trusted_registries = format_rules(config.fetch('trusted-registries', ''), 'trusted-registry')
+
+          [
+            'if ! command -v hadolint &> /dev/null; then',
+            '# Instead of installing hadolint in a step, ',
+            '# we recommend your agent environment to have it pre-installed',
+            'echo "~~~ Installing Hadolint"',
```

Comment:
this is part of the previous `if` statement so it should be indented one level deeper:

```suggestion
            '  # Instead of installing hadolint in a step, ',
            '  # we recommend your agent environment to have it pre-installed',
            '  echo "~~~ Installing Hadolint"',
```